packages/main/test/pages/Dialog.html

```diff
@@ -271,8 +273,8 @@
 		</div>
 	</ui5-dialog>
 
-	<ui5-dialog id="draggable-dialog" draggable>
-		<div slot="header" style="flex: 1 0 0; height: 2.5rem;">Draggable dialog</div>
+	<ui5-dialog id="draggable-dialog" accessible-name-id="tlt" draggable>
```

Comment:
Just as an option. 
Instead of providing the proposed API, could you add private "ariaLabelledBy" property and use it the same way as "accessibleNameId".

aria-labelledby is a standard attribute, so we can declare it in the metadata as private not to end up in the documentation, but at the same time we have it in the component's state. We used this approach in the Button for example.

```js

 * @private
 * @since 1.0.0-rc.15
ariaLabelledby: {
			type: String,
			defaultValue: "",
},
```
	
	
```html
	<ui5-dialog aria-labelledby="tlt">
```

Reply:
Thanks for the tip, but if it added with aria-labelledby, jaws does double readout with the combination of getEffectiveAriaLabelText. With the approach of the accessible-name validator and screen reader is happy.